rpc-darpc/src/main/java/com/ibm/crail/namenode/rpc/darpc/DaRPCServiceDispatcher.java

```diff
@@ -41,16 +41,17 @@
 	private static final Logger LOG = CrailUtils.getLogger();
 	
 	private RpcNameNodeService service;
+
+	protected AtomicLong totalOps;
+	protected AtomicLong createOps;
+	protected AtomicLong lookupOps;
+	protected AtomicLong setOps;
+	protected AtomicLong removeOps;
+	protected AtomicLong renameOps;
+	protected AtomicLong getOps;
+	protected AtomicLong locationOps;
+	protected AtomicLong errorOps;
```

Comment:
I prefer putting the Atomics in the stats class instead of making them protected